packages/kbn-code-owners/src/file_code_owner.ts

```diff
@@ -66,3 +67,31 @@ export function getCodeOwnersForFile(
 
   return match?.teams;
 }
+
+/**
+ * Run the getCodeOwnersForFile() method above.
+ * Report back to the cli with either success and the owner(s), or a failure.
+ *
+ * This function depends on a --file param being passed on the cli, like this:
+ * $ node scripts/get_owners_for_file.js --file SOME-FILE
+ */
+export async function runGetOwnersForFileCli() {
+  run(
+    async ({ flags, log }) => {
+      const targetFile = flags.file as string;
+      if (!targetFile) throw createFlagError(`Missing --flag argument`);
```

Comment:
Should we check if provided argument is actually a valid existing path?

Did some testing and I would expect error pointing the path is incorrect for the last 2 calls:

```
[get-owners-for-files][~/github/kibana]$ node scripts/get_owners_for_file.js --file packages/kbn-ace/src/ace/modes/index.ts
 succ elastic/kibana-management
[get-owners-for-files][~/github/kibana]$ node scripts/get_owners_for_file.js --file packages/kbn-ace/src/ace/modes
 succ elastic/kibana-management
[get-owners-for-files][~/github/kibana]$ node scripts/get_owners_for_file.js --file packages/kbn-ace/src/ace/mode
 succ elastic/kibana-management
[get-owners-for-files][~/github/kibana]$ node scripts/get_owners_for_file.js --file packages/kbn-ace/src/ace/x
 succ elastic/kibana-management
```

Reply:
Hrmm, as I understand it, the codewoners resolution is essentially looking for files and folders.  
Such that, if a file is not found, the result can be a parent dir (depending how the ownership is declared [specific file vs folder]).  This is my guess on the algo; that's how I do it manually.

That said, `$ node scripts/get_owners_for_file.js --file packages/kbn-ace/src/ace/x`:
```
 succ elastic/kibana-management
```
...gives me pause.  

Lemme see what I can discover.

Good catch mate!

Reply:
[FIx](https://github.com/elastic/kibana/pull/193277/commits/ea3b979435f4128d040787e6694ebc16c2c36bb8)?

Sorry about the force push.  I was in a meeting showing someone something and ran a rebase in the wrong dir.  Wont happen again (after opening pr's for review)

Reply:
The error message seems to be off. It mentions a `--flag` argument, that's not present in the args.